docs/src/components/components/action/link/README.md

```diff
@@ -0,0 +1,18 @@
+
+# Link
+
+The **link** component is used when you want the user to navigate to another
+page or document.
+
+## Use
+
+To use the **link** component,  add the classes `action action-link` to the `a`
+element.
+
+| Class   | Description |
+| ------   | -----------|
+| `action-primary` | When to use |
+| `action-base`       | When to use |
+| `action-warning`  | When to use |
+| `action-cautious`    | When to use |
```

Comment:
`warning`, `primary`, `finish`, and `base` are the allowed values in the Action `style` prop. I think `cautious` can be nixed.